internal/otel/collector.go

```diff
@@ -29,6 +31,13 @@ type CollectorCfg struct {
 	ResourceID        string
 	Client            hcp.TelemetryClient
 	ForwarderEndpoint string
+	ExporterConfig    *config.ExportConfig
+}
+
+// OTLPExporterConfig holds the type and
```

Comment:
Don't thinks this is used?

Reply:
yeah, I thought I had removed it but must have forgotten